### PR TITLE
XW-514 | add imports and fix eslint warnings

### PR DESCRIPTION
**Pull request by tomasznapieralski**

### CODE REVIEW

File: front/scripts/main/components/local-wikia-search.js [hunk 1]
```diff
@@ -126,7 +127,7 @@ const LocalWikiaSearchComponent = Ember.Component.extend({
 	 * @returns {string}
 	 */
 	getSearchURI(query) {
-		return `${get('apiBase')}/search/${encodeURIComponent(query)}`;
+		return `${App.get('apiBase')}/search/${encodeURIComponent(query)}`;
```
Comment (rogatty) [inline front/scripts/main/components/local-wikia-search.js]: Wow... Are we sure that it didn't happen anywhere else?

Comment (hakubo) [inline front/scripts/main/components/local-wikia-search.js]: checked - manually :)

File: front/scripts/main/CurrentUser.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import Ember from 'ember';
```
Comment (rogatty) [inline front/scripts/main/CurrentUser.js]: `current-user.js`?

File: front/scripts/main/models/article-add-photo.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import Ember from 'ember';
+import ArticleEditMixin from '../mixins/article-edit';
```
Comment (rogatty) [inline front/scripts/main/models/article-add-photo.js]: Sometimes there is `.js`, sometimes there's not. We should unify it.

**Pull request merged**